### ISSUE TITLE
Test plan review page (P0)

### ISSUE THREAD

**Issue by lolaodelola** [w3c/aria-at-app#679]
_Related to #648: Single Page Test Plans._ 

[This view of the alert plan](https://aria-at.netlify.app/review/alert.html) is provided as an example of a single-page view of a plan. Note that:

1. The link needs to be able to target a specific commit. The version of the plan that is "R&D Complete" will always be at the head of the main branch. However, if there is an earlier version of that plan that is in a later phase, it will be at an earlier point in the revision history.
2. There is a last edited date for each test that is not accurate.
    - It is showing the date and time of the latest commit to main instead of the date and time of revision of that plan.
    - The position of the last edited date implies that we can track when each test was edited, but we can't. We only know the date of the plan revision.
3. Changes to that single-page view should include:
    - Remove last edited from the individual tests.
    - Add an H2 with "Version created on DATE_TIME" under the H1 plan title.
    - Add a paragraph with "Summary of change: COMMIT_TITLE" followed by a div or paragraph containing the commit description.

**Comment by ccanash** [comment]
For context, the single page view of the test plan will be displayed in table within the Data Management page in the following places: 

**In the R&D Version Column** 

-  If the latest version has completed R&D and has not yet been advanced to draft review, show: Link with text "VERSION_STRING" that targets the single-page view of the plan. Example: [V22.12.08](https://aria-at.netlify.app/review/alert.html)

**In the Draft Review Column** 

- If there is a version currently in the draft phase, show: Link with text "VERSION_STRING" that targets the single-page view of the plan. Example: [V22.12.08](https://aria-at.netlify.app/review/alert.html)

**In the Candidate Review Column** 

- If there is a plan version currently in the candidate phase, show: Link with text "VERSION_STRING" that targets the single-page view of the plan. Example: [V22.12.08](https://aria-at.netlify.app/review/alert.html)In the 

**Recommended Version Column** 

- If there is a version of the plan currently in the recommended phase, show a Link with text "VERSION_STRING" that targets the single-page view of the plan. Example: [V22.12.08](https://aria-at.netlify.app/review/alert.html)

**Comment by mcking65** [comment]
@howard-e @ccanash 

Looking at the single page view of the plans in staging, tester instructions are missing information.

For example, read the NVDA tester instructions for test 1 in [Test plan review for pattern: disclosure-navigation](https://aria-at-staging.w3.org/test-review/179ba0f438aaa5781b3ec8a4033d6bf9f757360b/disclosure-navigation). They say:

>Tester Instructions
>1. 
>2. With the reading cursor on the 'Navigate forwards from here' link,navigate to the 'About' disclosure button. using the following commands: 
>* No commands have been added for this test for NVDA. Add commands, or, update the "applies_to" to list only the relevant assistive technologies. 

Then look at the [report](https://aria-at-staging.w3.org/report/1870/targets/197) for NVDA generated with this plan. It shows output and results for 7 commands.

**Comment by mcking65** [comment]
@ccanash per discussions this morning, Here are three additional requirements for this page:

1. Add the app navigation banner at the top so the page is branded and users can get to other pages in the app.
2. Under the H2 with "Version created on: VERSION_DATE" add information about working mode progress.
    * If current phase is R&D Complete, add element with string "Proposed  draft, review not yet started"
    * If draft review has started, replace prior string with "Draft review started on START_DATE"
    * If candidate review has started, add an additional element with string "Candidate review started on START_DATE" (be sure to create list format at this point)
    * If the version is recommended, add element with string "Approved as recommended on RECOMMENDED_DATE"
    * If version is deprecated, add element with string "Deprecated on DEPRECATION_DATE"
3. If the version is deprecated, also add "Deprecated " to the start of the H2 before the word "Version" and do something to the styling to emphasize the fact that this version of the plan is currently deprecated.

**Comment by mcking65** [comment]
Testing in staging, overall, this page is looking great with a few exceptions , none of which should block deployment or closure of this issue. I have opened the following issues to track the exceptions.

There is one but realated to the title tag and H1. I raised #789.

There are some requirements listed in [my previous comment](https://github.com/w3c/aria-at-app/issues/679#issuecomment-1662861833) that have not been addressed. I raised #788.
